src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import {SafeAreaView, View, Image} from 'react-native'
+
+import SCREENS from '../constants/screens'
+import {containerStyles, colors} from '../styles'
+import {Button, Link, PageHeader} from '../components'
+
+function Login({navigation}: any) {
+  return (
+    <SafeAreaView
+      style={[containerStyles.fill, {backgroundColor: colors.white}]}>
+      <View
+        style={[
+          containerStyles.fill,
+          containerStyles.centeredContent,
+          containerStyles.pageContainer,
+        ]}>
+        <Image
+          style={[
+            {
+              width: '90%',
```

Comment:
Ok - the problem here is that you only have 1 image asset for the passport. You need to replace it with all of the exported sizes from here:

https://drive.google.com/drive/folders/1F7p1MvmK6zv0MlkeK4Ih5r2g93uU7kzE

Then your image sizing issues will resolve themselves and you won't need to include any height for the image.